### PR TITLE
Instructions on how to deploy a branch to CG-VM1 in a collapsible section of PR template

### PR DESCRIPTION
**Pull request by northwestwitch**
Adds instructions on how to deploy test branches to cg-vm1 

**Review:**
- [x] code approved by DN
- [x] tests executed by CR, DN

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -2,6 +2,16 @@ This PR adds a functionality or fixes a bug.
 OR
 This PR marks a new Scout release. We apply semantic versioning. This is a major/minor/patch release for reasons.
 
+**Prepare for testing**
```
Comment (henrikstranneheim) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: Maybe add a Stockholm node header?

Comment (northwestwitch) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: Good suggestion, I'll add it!

**Comment by northwestwitch** [comment]
Final final change: since having too many instructions when describing a new PS is painful, I've included these instructions in a collapsible section of the comment. It will look like this:

image


That expanded become like this:

image

**Comment by dnil** [comment]
Note that it should normally be possible to stop and start the service with 
`systemctl stop scout.target`
It might be that scout missing from the everything.target does that (https://github.com/Clinical-Genomics/servers/pull/735), or its just something with shell escaping that is acting weirdly..

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -2,6 +2,20 @@ This PR adds a functionality or fixes a bug.
...
+<details>
+<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>
+
+**Prepare for testing**
+1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
+1. `ssh your.email@cg-vm1.scilifelab.se`
+1. `sudo -iu hiseq.clinical`
+1. `ssh localhost`
+1. Find which scout branch (old_branch) is currently deployed on cg-vm1: `podman ps`
+1. Stop the service with current deployed branch: `systemctl --user stop scout@<old_branch>`
```
Comment (dnil) [inline .github/PULL_REQUEST_TEMPLATE.md:14]:
```suggestion
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -2,6 +2,20 @@ This PR adds a functionality or fixes a bug.
...
 
+<details>
+<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>
+
+**Prepare for testing**
+1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
+1. `ssh your.email@cg-vm1.scilifelab.se`
+1. `sudo -iu hiseq.clinical`
+1. `ssh localhost`
+1. Find which scout branch (old_branch) is currently deployed on cg-vm1: `podman ps`
```
Comment (dnil) [inline .github/PULL_REQUEST_TEMPLATE.md:13]:
```suggestion
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
```

**Comment by northwestwitch** [comment]
> Note that it should normally be possible to stop and start the service with
> `systemctl stop scout.target`

It does work, but with the --user thinghy, I could write that instruction instead, but I was thinking that perhaps it's useful for the user to know which branch is deployed on scout stage before starting testing, to understand for instance if somebody else is testing in that moment?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -2,6 +2,20 @@ This PR adds a functionality or fixes a bug.
...
+1. Stop the service with current deployed branch: `systemctl --user stop scout@<old_branch>`
+1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
+1. Make sure the branch is deployed: `systemctl --user status scout@<this_branch>`
```
Comment (dnil) [inline .github/PULL_REQUEST_TEMPLATE.md:16]:
```suggestion
1. Make sure the branch is deployed: `systemctl --user status scout.target`
```

**Review by dnil — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (northwestwitch) [reply .github/PULL_REQUEST_TEMPLATE.md]: Good solution! 👍🏻

**Pull request merged**